services/ui-src/src/libs/detailLib.ts

```diff
@@ -110,19 +118,31 @@ export const defaultPackageOverviewNavItems: DetailNavItem[] = [
 export const defaultPackageOverviewLabel: string = "Package Overview";
 
 export const defaultDetailSectionItems = [
-  typeDefault,
+  submissionIdDefault,
+  blankBox, // empty space
   territoryDefault,
+  typeDefault,
   submissionDateDefault,
-  latestRaiResponseDateDefault,
   proposedEffectiveDateDefault,
+  latestRaiResponseDateDefault,
+  blankBox, // empty space
```

Comment:
Could we use a nested array here as the type definition of a "SectionItems"? Whereby the first array is the first column and the second array is the second column. That way we could define something like:
export const defaultDetailSectionItems = 
[
  [field1, field2, field3, field4],
  [blankBox, field 5]
 ]
 
 i suppose the downside is we are building layout into the config when thats the job of the page......but id argue we already made that choice to genericize it by enforcing layout with the order of the detail items in the array. This setup would lead nicely into how you are already using a double nested array in the detail component and would also get rid of the moving column issue for unrendered data in column A.

Reply:
Hey! There's actually going to be another unit of work (see [Erika's comment](https://qmacbis.atlassian.net/browse/OY2-24007?focusedCommentId=121141) on JIRA) that will put the layout strictly in the hands of the component. After talking with Erika, we determined the AC for this work would be to simply take what we have now, and make it two-columns and "as close as we can to the Figma" -- in this case, the divider, Description being a full-width-single-column, and some empty spaces were cut until we can layout the page based on an object with defined attributes and not the array we have now.

As for the permissions, I don't _think_ any of that code got removed...that's odd, and I'm looking into it

Reply:
Woops -- those fields just didn't have a role restriction applied to them is all. Tho...that's curious, because if they _don't_ show now for State users in Dev/Val and they didn't have the `rolePrivilate` attribute, how were they hidden? lol

Reply:
sounds good! I wish we could get ACs updated by business to reflect discussions in the comments..... and yeah i couldnt see why your changes would have changed the permissions on those fields either.
Thanks!